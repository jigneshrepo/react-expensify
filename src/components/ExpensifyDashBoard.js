import React, {Component} from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters'

class ExpensifyDashBoard extends React.Component{
    render(){
        return (
            <div>
                <ExpenseListFilters/>
                <ExpenseList/>
            </div>
        );
    }
}

export default ExpensifyDashBoard;