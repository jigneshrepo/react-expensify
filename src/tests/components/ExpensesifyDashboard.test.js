import React from 'react';
import {shallow} from 'enzyme';
import ExpensifyDashBoard from '../../components/ExpensifyDashBoard';

test('should render dashboard correclty',()=>{
    const wrapper = shallow(<ExpensifyDashBoard />);
    expect(wrapper).toMatchSnapshot();
});
