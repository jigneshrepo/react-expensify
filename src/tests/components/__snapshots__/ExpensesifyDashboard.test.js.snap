// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard correclty 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(Component) />
</div>
`;
